src/Row.ts

```diff
@@ -37,15 +37,44 @@ class Row extends RowBase<RowProperties> {
 				classes: this.classes(tableCss.table, css.rowTable)
 			}, [
 				v('tr', columns.map((column) => {
-					const { id, field } = column;
+					const { field, id } = column;
+
+					let value: any;
+					if (column.get) {
+						// Get the value from the column callback
+						value = column.get(item, column);
+					}
+					else if (!column.render) {
+						// Get the value using a property lookup on the item data
+						value = item.data[ field || id ];
+					}
+					else {
+						// This column has no value
+						value = '';
+					}
+
+					let content: DNode;
+					if (column.render) {
```

Comment:
I see you have updated the API, to normalise the `render` functions, I would still consider a different name than `render` (to remove potential confusion)